torch/csrc/autograd/python_variable.cpp

```diff
@@ -751,9 +751,9 @@ static PyObject* THPVariable_make_wrapper_subclass(
       "SymInt? storage_offset=None, MemoryFormat? memory_format=None, ScalarType dtype=None, "
       "Layout layout=torch.strided, Device device=None, bool pin_memory=False, bool requires_grad=False, "
       "c10::string_view? dispatch_sizes_strides_policy=None, bool dispatch_device=False, bool dispatch_layout=False, "
-      "DispatchKeySet _extra_dispatch_keys=None)",
+      "DispatchKeySet _extra_dispatch_keys=None, SymInt? _override_storage_size=None)",
```

Comment:
```suggestion
      "DispatchKeySet _extra_dispatch_keys=None, SymInt? storage_size=None)",
```

- private arguments is not a thing, everything is visible to the user in the signature.
- We need to update the doc to explain what this argument is doing: set the storage size, if not provided, we use the smallest storage size possible based on the other arguments

Reply:
@albanD

> update the doc to explain what this argument is doing

where is this doc? I couldn't find any docstring for this

Reply:
Ho wow, I assumed we had a docstring or a cpp comment for this :o 
Well, I guess that's fine, feel free to leave a small comment in cpp above the function saying what this does as a start